.travis.yml

```diff
@@ -1,26 +1,15 @@
 language: python
 python:
-  - 3.5
   - 3.6
 
 env:
-  - TOXENV=py35
-  - TOXENV=py36
-  - TOXENV=flake8
-  - TOXENV=docs
+  global:
+    - DOCKER_COMPOSE_VERSION=1.19.0
+  matrix:
+    - TOXENV=py36
+    - TOXENV=flake8
+    - TOXENV=docs
   
-matrix:
```

Comment:
un-related to this PR by why we dont support python 3.5 now? because thats still the default for ubuntu-xenial and other latest OSes I assume? cc @vrde

Reply:
We support only Python 3.6. I quickly talked with @muawiakh about that. We can drop `tox` for now IMO.

Reply:
Yes, I will remove it from this PR. If we see a problem with the driver while testing with `3.5` we can deprecate the support. I will revert this change for now.